### PR TITLE
Introduce a conda-forge recipe for GWSumm

### PR DESCRIPTION
**Pull request by alurban**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages
- [X] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by alurban** [comment]
Note, @duncanmmacleod has expressed to me privately that he consents to this release.

**Comment by duncanmmacleod** [comment]
@alurban why are we skipping py37?

**Comment by alurban** [comment]
@duncanmmacleod, it's not explicitly supported by the package: https://pypi.org/project/gwsumm/. I debated adding it to the setup.py file before releasing, but thought you might've had a good reason for not including it.

**Comment by duncanmmacleod** [comment]
@alurban, I think its just that the classifiers are out of date, I would say that if it builds on py37, use it.

**Comment by alurban** [comment]
@duncanmmacleod, I think I found the root of the actual build failure, which is that I was pointing to the wrong place to look for static CSS/JS files. I'll try again with python-3.7.

**Comment by alurban** [comment]
@conda-forge/help-python, this PR is now ready for review.

**Comment by alurban** [comment]
@duncanmmacleod, @conda-forge/help-python, I've updated this recipe to point to gwsumm-0.1.1, a point release that fixes some python-3.7 compatibility issues.

File: recipes/gwsumm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+requirements:
+  host:
+    - jsmin
+    - libsass
+    - pip
+    - python
+    - setuptools
+  run:
+    - python >=2.7
```
Comment (chrisburr) [inline recipes/gwsumm/meta.yaml]: As this isn't `noarch` you shoudn't constrain the python version:
```suggestion
    - python
```

File: recipes/gwsumm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    - gw_summary_pipe --help
+    - gwsumm-plot-guardian --help
+    - gwsumm-plot-triggers --help
+    # check HTML files were built and installed
+    - test -f ${SP_DIR}/gwsumm/html/static/gwsumm.min.css
+    - test -f ${SP_DIR}/gwsumm/html/static/gwsumm.min.js
+
+about:
+  home: https://github.com/gwpy/gwsumm
+  license: GPLv3
```
Comment (chrisburr) [inline recipes/gwsumm/meta.yaml]: From https://opensource.org/licenses/alphabetical:
```suggestion
  license: GPL-3.0
```

**Comment by alurban** [comment]
Thanks for getting back so quickly @chrisburr, I've addressed your two comments so far

**Pull request merged**